### ISSUE TITLE
Deleting Pod From Nodepool Deletes Nodepool and Not Pod

### ISSUE THREAD

**Issue by gvanriper** [kontena/lens#5521]
**Describe the bug**
When navigating from nodepool to your pod resource and deleting that pod, it is removing the entire nodepool instead of the pod. This used to only remove the pod itself.

**To Reproduce**
1. Select Nodepool
2. Select a pod running in that Nodepool.
3. Select delete pod.
4. Deleting pod will now have removed the nodepool.

**Expected behavior**
Deleting should only delete the pod, not the entire nodepool.

**Environment (please complete the following information):**
- Lens Version:
![Screen Shot 2022-05-31 at 4 49 01 PM](https://user-images.githubusercontent.com/8250434/171282024-59c0998b-84bf-4058-8531-ec903f7fb8bb.png)

I am on Mac OS  Monterey 12.4

- Installation method (e.g. snap or AppImage in Linux):
This happened during the automatic update of Lens. My initial install was using the dmg file on the k8slens.com site.

**Logs:**
N/A

**Kubeconfig:**

**Note: I have to redact most things...**
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: [REDACTED]
    server:[REDACTED]
  name: [REDACTED]
- cluster:
    certificate-authority-data: [REDACTED]
    server: [REDACTED]
  name:[REDACTED]
- cluster:
    certificate-authority-data: [REDACTED]
    server: [REDACTED]
  name: [REDACTED]
- cluster:
    certificate-authority-data: [REDACTED]
    server: [REDACTED]
  name: [REDACTED]
- cluster:
    certificate-authority-data: [REDACTED]
    server: [REDACTED]
  name: [REDACTED]
- cluster:
    certificate-authority-data: [REDACTED]
    server: [REDACTED]
  name: [REDACTED]
- cluster:
    certificate-authority-data: [REDACTED]
    server: [REDACTED]
  name: [REDACTED]
- cluster:
    certificate-authority-data: [REDACTED]
    server: [REDACTED]
  name: [REDACTED]
contexts:
- context:
    cluster: [REDACTED]
    user: clusterUser_[REDACTED]
  name: [REDACTED]
- context:
    cluster: [REDACTED]
    user: clusterUser_[REDACTED]
  name: [REDACTED]
- context:
    cluster: [REDACTED]
    user: clusterUser_[REDACTED]
  name: [REDACTED]
- context:
    cluster: [REDACTED]
    user: clusterUser_[REDACTED]
  name: [REDACTED]
- context:
    cluster: docker-desktop
    user: docker-desktop
  name: docker-desktop
- context:
    cluster: [REDACTED]
    user: clusterUser_[REDACTED]
  name: [REDACTED]
- context:
    cluster: [REDACTED]
    user: clusterUser_[REDACTED]
  name: [REDACTED]
- context:
    cluster: [REDACTED]
    user: clusterUser_[REDACTED]
  name: [REDACTED]
current-context: [REDACTED]
kind: Config
preferences: {}
users:
- name: clusterUser_[REDACTED]
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
    token: [REDACTED]
- name: clusterUser_[REDACTED]
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
    token: [REDACTED]
- name: clusterUser_[REDACTED]
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
    token: [REDACTED]
- name: clusterUser_[REDACTED]
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
    token: [REDACTED]
- name: clusterUser_[REDACTED]
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
    token: [REDACTED]
- name: clusterUser_[REDACTED]
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
    token: [REDACTED]
- name: docker-desktop
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
- name: clusterUser_[REDACTED]
  user:
    client-certificate-data: [REDACTED]
    client-key-data: [REDACTED]
    token: [REDACTED]

```

**Additional context**
This did not happen until last week during a recent update.

Our environment is utilizing Azure AKS clusters.

**Comment by Tomasz-Kluczkowski** [comment]
I just realised I got bitten by this! Was not sure what the hell happened. Indeed selecting a pod from within the node's workload and clicking delete, deletes the node, instead of the pod which is currently focused.

My Lens is:
![Screenshot 2022-06-01 at 13 19 22](https://user-images.githubusercontent.com/26039401/171402908-0f994348-0ddd-4965-8af1-c091ad3f7005.png)

Lens: 5.5.2-latest.20220530.1

**Comment by gvanriper** [comment]
> I just realised I got bitten by this! Was not sure what the hell happened. Indeed selecting a pod from within the node's workload and clicking delete, deletes the node, instead of the pod which is currently focused.
> 
> My Lens is: ![Screenshot 2022-06-01 at 13 19 22](https://user-images.githubusercontent.com/26039401/171402908-0f994348-0ddd-4965-8af1-c091ad3f7005.png)

Yeah... I got burnt pretty hard myself. Brought my web app down for a bit and I didn't notice what had happened until I did it again. Doh!

**Comment by eugenesotnikov** [comment]
+1

**Comment by Nokel81** [comment]
Duplicate of #5492 , will be fixed by #5505

**Comment by gvanriper** [comment]
> Duplicate of #5492 , will be fixed by #5505

Yes, sorry. I swear I tried to search for a dupe but I didn't come up with anything. Glad you got it fixed!

**Comment by Nokel81** [comment]
All good
